### PR TITLE
Role badge + affiliation for contributors

### PR DESCRIPTION
**Pull request by bedroesb**
After this PR editors will have the `Editor` badge in the contributors page + it is possible to specify you affiliation in the [CONTRIBUTORS](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml) file

Screenshot from 2021-04-01 20-26-27

Screenshot from 2021-04-01 20-26-55

Should we do an effort in updating the existing contributors?

### CODE REVIEW

File: _data/CONTRIBUTORS.yaml [hunk 1]
```diff
@@ -6,32 +6,44 @@ Bert Droesbeke:
...
+    role: editor
+    affiliation: VIB-UGent
 Mijke Jetten:
     git: MijkeJetten
     email: m.jetten@dtls.nl
     orcid: 0000-0001-9114-2896
+    role: editor
 Nicola Soranzo:
     git: nsoranzo
     orcid: 0000-0003-3627-5340
```
Comment (nsoranzo) [inline _data/CONTRIBUTORS.yaml:36]:
```suggestion
    orcid: 0000-0003-3627-5340
    affiliation: Earlham Institute
```

**Comment by bedroesb** [comment]
with my last commit you can have a custom role in the badge + a custom picture if people are not on Github:

Screenshot from 2021-04-02 12-15-01

**Comment by nsoranzo** [comment]
You may also consider https://en.gravatar.com/ for profile images.

**Comment by bedroesb** [comment]
@nsoranzo I will add this suggestion to our to/idea list #447

**Pull request merged**